wlog/auditlog/audit3log/categories.go

```diff
@@ -0,0 +1,785 @@
+// Copyright (c) 2025 Palantir Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by category_transform.go using source file version 6.101.0, DO NOT EDIT.
```

Comment:
As noted here, this is a generated file, so every line does not need to be reviewed.

However, it would be helpful to do a spot check to ensure that the general logic that is generated makes sense/is correct.